checkov/main.py

```diff
@@ -353,6 +353,9 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
             file = config.file
             scan_reports = runner_registry.run(root_folder=root_folder, external_checks_dir=external_checks_dir,
                                                files=file)
+            is_error = any([scan_report.error_status for scan_report in scan_reports])
```

Comment:
```suggestion
            is_error = any(scan_report.error_status for scan_report in scan_reports)
```
if I'm not mistaken, then this should also work and you save the time for creating a list 🙂

Reply:
yep you are right =] 
will be fixed